torch/csrc/jit/operator.cpp

```diff
@@ -227,11 +227,37 @@ using OperatorMap = std::unordered_map<Symbol, std::vector<std::shared_ptr<Opera
 struct OperatorRegistry  {
   OperatorMap operators;
   std::mutex lock;
+  // TODO: comment
+  std::unordered_map<std::string, std::shared_ptr<Operator>> operators_by_sig;
+  std::unordered_map<const char *, std::shared_ptr<Operator>> operators_by_sig_literal;
```

Comment:
That's true but I think being strict is a benefit. For example, when a signature changes you only need to do a very simple codemod, because you're guaranteed that all matches are going to look in a very particular way. We can surely make it more robust wrt. some other annotations later but I think this is a better way to go for now.